clarity/evaluator/msbg/msbg_utils.py

```diff
@@ -105,167 +105,185 @@ def read_gtf_file(gtf_file):
     # Fix filename if necessary
     dirname = os.path.dirname(os.path.abspath(__file__))
     gtf_file = os.path.join(dirname, gtf_file)
-    with open(gtf_file, "r", encoding="utf-8") as fp:
-        data = json.load(fp)
```

Comment:
I'm a bit conflicted about this one: I've seen 'fp' used for "file pointer" in these short lived contexts in lots of pro code - I feel it is more conventional than "file_pointer" which seems a bit cumbersome.  If we have multiple files open we'd want to name the pointers e.g. "input_fp" and "output_fp" ... this would be more readable to me than "input_file_pointer" and "output_file_pointer".   

Unless you feel strongly otherwise, I suggest reverting to 'fp' but adding  "fp" to the "good-names' list in .pylintrc -- we are at least accepting it actively and documenting it as a list of short names we allow..

Reply:
Done and thanks for the pointer to `good-names` in `.pylintrc`, not something I was aware of but will certainly be leveraging from now on.